src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/MemoryCache.cs

```diff
@@ -21,10 +21,10 @@ internal class MemoryCache<TKey, TValue>
         private readonly object _compactLock;
         private readonly int _sizeLimit;
 
-        public MemoryCache(int sizeLimit = DefaultSizeLimit)
+        public MemoryCache(int sizeLimit = DefaultSizeLimit, int concurrencyLevel = 2)
```

Comment:
Nit: I like parameterizing this, maybe make the 2 `private const int DefaultConcurrencyLevel = 2;` just to match the other parameter?